screens/HotspotFormScreen.tsx

```diff
@@ -257,6 +258,18 @@ export const HotspotFormScreen = ({
     load();
   }, []);
 
+  useEffect(() => {
+    if (location)
+      setHotspotDetails({
+        ...hotspotDetails,
+        city: location.city || '',
```

Comment:
is there a problem if we leave these `undefined`?

Reply:
These are required fields